external-docs/docs/guides/linked-data/organisations.md

```diff
@@ -0,0 +1,1139 @@
+|[Financial Remedies Court](https://www.gov.uk/government/organisations/financial-remedies-court)|<https://www.gov.uk/government/organisations/financial-remedies-court>|
+|[Flood and Coastal Erosion Risk Management Research and Development Programme](https://www.gov.uk/government/organisations/flood-and-coastal-erosion-risk-management-research-and-development-programme)|<https://www.gov.uk/government/organisations/flood-and-coastal-erosion-risk-management-research-and-development-programme>|
+|[Government Data Quality Hub](https://www.gov.uk/government/organisations/government-data-quality-hub)|<https://www.gov.uk/government/organisations/government-data-quality-hub>|
+|[Government Skills and Curriculum Unit](https://www.gov.uk/government/organisations/government-skills-and-curriculum-unit)|<https://www.gov.uk/government/organisations/government-skills-and-curriculum-unit>|
+|[Independent Monitoring Authority for the Citizens’ Rights Agreements](https://www.gov.uk/government/organisations/independent-monitoring-authority-for-the-citizens-rights-agreements)|<https://www.gov.uk/government/organisations/independent-monitoring-authority-for-the-citizens-rights-agreements>|
+|[Office for Zero Emission Vehicles](https://www.gov.uk/government/organisations/office-for-zero-emission-vehicles)|<https://www.gov.uk/government/organisations/office-for-zero-emission-vehicles>|
+|[Salix Finance Ltd](https://www.gov.uk/government/organisations/salix-finance-ltd)|<https://www.gov.uk/government/organisations/salix-finance-ltd>|
+|[Equality Hub](https://www.gov.uk/government/organisations/the-equality-hub)|<https://www.gov.uk/government/organisations/the-equality-hub>|
+|[Trade Remedies Authority](https://www.gov.uk/government/organisations/trade-remedies-authority)|<https://www.gov.uk/government/organisations/trade-remedies-authority>|
+|[UK Health Security Agency](https://www.gov.uk/government/organisations/uk-health-security-agency)|<https://www.gov.uk/government/organisations/uk-health-security-agency>|
+|[UK National Authority for Counter-Eavesdropping](https://www.gov.uk/government/organisations/uk-national-authority-for-counter-eavesdropping)|<https://www.gov.uk/government/organisations/uk-national-authority-for-counter-eavesdropping>|
+|[FCDO Services ](https://www.gov.uk/government/organisations/fcdo-services)|<https://www.gov.uk/government/organisations/fcdo-services>|
+|[UK National Screening Committee](https://www.gov.uk/government/organisations/uk-national-screening-committee)|<https://www.gov.uk/government/organisations/uk-national-screening-committee>|
+|[Youth Custody Service](https://www.gov.uk/government/organisations/youth-custody-service)|<https://www.gov.uk/government/organisations/youth-custody-service>|
+
+## Adding an organisation
+
+There is not currently an automatic way to add a valid organisation. If your organisation is not on the above list and you would like it to be added, please [raise an issue](../raise-issue.md).
```

Comment:
People can claim their dataset was published/created by anyone. Just like you wouldn't trust a PDF posted on `http://xsdasd.dodgy.url.com/hmg.pdf` that claimed to be a publication of HMG, you shouldn't trust a CSV-W that is hosted somewhere non-official.

Reply:
Perhaps we should create a ticket to look at being able to cryptographically sign CSV-Ws that you've generated so that other users can validate that the CSV-W genuinely was produced by the right orgainsation.